src/clj_momo/metrics/console.clj

```diff
@@ -0,0 +1,9 @@
+(ns clj-momo.metrics.console
+  (:require [metrics.reporters.console :as console]
+            [ctia.properties :refer [properties]]))
```

Comment:
ctia.properties won't work, I don't think we can pass in ctim.properties either (the atom isn't defined in clj-momo).  Need to pass in an fn, or somthing.